docs/api-reference.md

```
@@ -0,0 +1,79 @@
+# API Reference
+
+Base URL: `/`
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)